spec/features/creating_messages_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'timecop'
+
+feature 'adding a new message' do
+  scenario 'a user can add a message to Chitter' do
+    Timecop.freeze(Time.local(2022, 7, 15)) do
```

Comment:
Cool! Timecop is useful. Another option that doesn't require a gem would have been to stub the return value of `Time.now`, e.g.

```
@date = Time.local(2022, 7, 15)
allow(Time).to receive(:now).and_return(@date)
```